src/corelibs/U2View/src/ov_sequence/AnnotationsTreeView.cpp

```diff
@@ -674,51 +674,51 @@ void AnnotationsTreeView::sl_onAnnotationsRemoved(const QList<Annotation *> &as)
-            QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
-            foreach (AVAnnotationItem *ai, aItems) {
-                ai->removeQualifier(qm.getQualifier());
-            }
-        } break;
-        case AnnotationModification_QualifierAdded: {
-            const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
-            QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
-            foreach (AVAnnotationItem *ai, aItems) {
-                if (ai->isExpanded() || ai->childCount() > 1) {    //if item was expanded - add real qualifier items
-                    ai->addQualifier(qm.getQualifier());
-                } else {
-                    ai->setChildIndicatorPolicy(QTreeWidgetItem::ShowIndicator);    //otherwise process indicator only
+            case AnnotationModification_QualifierRemoved: {
+                const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
+                QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
+                foreach (AVAnnotationItem *ai, aItems) {
+                    ai->removeQualifier(qm.getQualifier());
                 }
-            }
-        } break;
-        case AnnotationModification_AddedToGroup: {
-            const AnnotationGroupModification &gmd = static_cast<const AnnotationGroupModification &>(annotationModification);
-            AVGroupItem *gi = findGroupItem(gmd.getGroup());
-            SAFE_POINT(NULL != gi, L10N::nullPointerError("annotation view group item"), );
-            buildAnnotationTree(gi, gmd.annotation);
-            gi->updateVisual();
-        } break;
+            } break;
+            case AnnotationModification_QualifierAdded: {
+                const QualifierModification &qm = static_cast<const QualifierModification &>(annotationModification);
+                QList<AVAnnotationItem *> aItems = findAnnotationItems(qm.annotation);
+                foreach (AVAnnotationItem *ai, aItems) {
+                    if (ai->isExpanded() || ai->childCount() > 1) {    //if item was expanded - add real qualifier items
+                        ai->addQualifier(qm.getQualifier());
+                    } else {
+                        ai->setChildIndicatorPolicy(QTreeWidgetItem::ShowIndicator);    //otherwise process indicator only
+                    }
+                }
+            } break;
+            case AnnotationModification_AddedToGroup: {
+                const AnnotationGroupModification &gmd = static_cast<const AnnotationGroupModification &>(annotationModification);
+                AVGroupItem *gi = findGroupItem(gmd.getGroup());
+                SAFE_POINT(NULL != gi, L10N::nullPointerError("annotation view group item"), );
+                buildAnnotationTree(gi, gmd.annotation);
+                gi->updateVisual();
+            } break;
```

Comment:
And here/ And everywhere it the same place

Reply:
Same here